etc/eb/.ebextensions/nginx/conf.d/01_ssl_proxy.conf

```diff
@@ -1,3 +1,11 @@
+map $host $frontend {
+  default "http://pension.tuleva.ee.s3-website.eu-central-1.amazonaws.com";
+  # default "http://192.168.65.2:3000";
```

Comment:
What is this IP? If local then use `127.0.0.1` instead. Maybe you can use this trick in the other nginx file which includes this file to have it as default https://stackoverflow.com/a/30083902

Reply:
Docker localhost